pkg/console/subresource/configmap/configmap.go

```diff
@@ -47,12 +47,13 @@ func DefaultConfigMap(
 	nodeOperatingSystems []string,
 	copiedCSVsDisabled bool,
 	telemeterConfig map[string]string,
+	consoleHost string,
```

Comment:
I see the point of adding this additional param, but I would rather avoid it, since for the non ALB setup, `consoleHost` will be the same as the host int the `activeConsoleRoute.spec`.
Lets rather create a new struct, which would include both of these types.

Reply:
> since for the non ALB setup, consoleHost will be the same as the host int the activeConsoleRoute.spec

Well, most of the time it's true. The `route.spec.host` is assigned automatically by the apiserver from the default ingress domain without the knowledge about the inrgesscontrollers which is only present in the route's status. That's why in `operator/sync_v400.go` I pass down the url returned by `routesub.GetActiveRouteInfo()` which returns only the admitted host.